_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+
+###### Time Management
+
+Teams should not be fully allocated to their enabling function.
+
+Firstly, each team must resolve time for the continuous self-improvement, which should be equal to, or strictly greater than, 20% of their teams available work time.
+
+Secondly, teams are free to self organize under the condition that they are able to meet the needs of the directorate’s priorities. For example, a team within the directorate may accept to fulfill an enabling function for another team within the organization.
+
+Thirdly, an enabling team need not commit all of their time to being an enabling team. For example, in the case of the IT Strategy team they are also responsible for setting the IT vision for the department, and thus cannot dedicate 100%(-20%) of their time to their enabling function. In this particular case, for example, we recommend that the IT Strategy spend a maximum of (100-20)/2 percent of their time on enabling other teams.
+
+##### External Interactions
+
+This is an initial proposal and is intended to allow for iterations to refine and modify the process as needed.
+
+###### Interactions
+
+The book Team Topologies, referenced below, recommends that Enabling Teams work with other teams for a period (potentially a few months) [TO-DO: Quote needed] in order to assist them with onboarding new technologies and approaches, and improving their existing processes and procedures. The goal is to ensure the interactions are brief enough that the teams being enabled do not become dependent on the enabling teams. The process improvements should stand on their own, and should not involve [toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/). That is to say, the new processes and procedures should aim to automate or improve the existing processes, not lend temporary person-power to the teams. Given the short and iterative nature of the interactions between the teams (duration of interactions should be defined by number of sprints) the enabling teams must identify quick wins -- the amount of automation that could be done within the limited time frame that would result in the greatest amount of automated work.
+
+###### Tooling
+
+[TO-DO: Quote from Team Topologies regarding varying levels of team communication]
+
+The amount of collaboration between the enabling teams and the enabled teams should vary significantly during the time which the enabling team is paired with the enabled team. As such, collaboration tools that allow for the rapid creation and disintegration should be leveraged. An example of a viable tool for such a requirement would be Slack, where groups can be created and disposed of with relative ease.
+
+#### Process
```

Comment:
Am I reading this correctly that this is an intake and WIP management process?
I.e. each ITSM teams (IT strategy, s/w, h/w, and IT service) have part of their time associated to enabling other teams. That means they have limitted capacity to intake enablement requests.

During the teams iterative process, each team will have to determine if they have capacity to take on new requests, or if their existing requests need to be re-prioritize.

Reply:
I think since we are enabling teams, we don't have as much "requests"? Like, we get the odd off the wall request to do X but usually our role is to help other teams like the Value Stream Aligned Teams, which deal with requests?

Reply:
Could be a value stream aligned team, could be another enabling team. Could be anyone who wants to work better.

The idea is requests come from two places, and the team must prioritize how they manage the work during their planning meetings. 

Work may come from the director, if the entire directorate is enabling another directorate, for example. 

Work may come from one manager to another manage within ITPSM. 

In either case, the team must prioritize the work accoringly. 

In the case where the work comes from the director, this is when the synchonrized sprints come into play, as it will be hard to unify and coordinate work between 3 teams if they all out of sync when they are working on the work handed down by the director.